v23.1/user-defined-functions.md

```diff
@@ -105,6 +92,192 @@ If you do not specify a schema for the function `add` when you create it, the de
+                KV gRPC calls: 1
+                estimated max memory allocated: 60 KiB
+                sql cpu time: 87µs
+                estimated row count: 1,000 (100% of the table; stats collected 19 seconds ago)
+                table: a@a_pkey
+                spans: FULL SCAN
+    (33 rows)
+    ~~~
+
+    The query takes `77ms` to execute because the function is evaluated for each row in `a`.
+
+1. View the query plan when using `foo_s()` (the `STABLE` function) instead:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    EXPLAIN ANALYZE SELECT foo_s(a) FROM a WHERE a = 10;
+    ~~~
+
+    ~~~
+                                                  info
+    ------------------------------------------------------------------------------------------------
+      planning time: 5ms
+      execution time: 4ms
+      distribution: local
+      vectorized: true
+      rows read from KV: 1,001 (39 KiB, 2 gRPC calls)
+      cumulative time spent in KV: 832µs
+      maximum memory usage: 420 KiB
+      network usage: 0 B (0 messages)
+      sql cpu time: 3ms
+      estimated RUs consumed: 0
+
+      • render
+      │
+      └── • merge join (left outer)
+          │ nodes: n1
+          │ actual row count: 1,000
+          │ estimated max memory allocated: 340 KiB
+          │ estimated max sql temp disk usage: 0 B
+          │ sql cpu time: 3ms
+          │ estimated row count: 1,000
+          │ equality: (a) = (b)
+          │ right cols are key
+          │
+          ├── • filter
+          │   │ nodes: n1
+          │   │ actual row count: 1,000
+          │   │ sql cpu time: 5µs
+          │   │ estimated row count: 1,000
+          │   │ filter: a = 10
+          │   │
+          │   └── • scan
+          │         nodes: n1
+          │         actual row count: 1,000
+          │         KV time: 722µs
+          │         KV contention time: 0µs
+          │         KV rows read: 1,000
+          │         KV bytes read: 39 KiB
+          │         KV gRPC calls: 1
+          │         estimated max memory allocated: 60 KiB
+          │         sql cpu time: 202µs
+          │         estimated row count: 1,000 (100% of the table; stats collected 42 seconds ago)
+          │         table: a@a_pkey
+          │         spans: FULL SCAN
+          │
+          └── • scan
+                nodes: n1
+                actual row count: 1
+                KV time: 110µs
+                KV contention time: 0µs
+                KV rows read: 1
+                KV bytes read: 30 B
+                KV gRPC calls: 1
+                estimated max memory allocated: 20 KiB
+                sql cpu time: 11µs
+                estimated row count: 1 (100% of the table; stats collected 42 seconds ago)
+                table: b@b_pkey
+                spans: FULL SCAN
+    (57 rows)
+    ~~~
+
+    The query takes only `4ms` to execute because the function is inlined as an equality comparison `(a) = (b)` with a much lower overhead.
```

Comment:
nit: maybe improve this with something like: "because the function is inlined **and transformed to a join with an equality comparison, which has much less overhead than invoking a function for each row scanned in table `a`.**